src/map.c

```diff
@@ -1385,6 +1389,12 @@ void map_renderNames( double bx, double by, double x, double y,
       col.a = alpha;
       gl_printRaw( font, tx, ty, &col, -1, _(sys->name) );
 
+      /* Render note */
```

Comment:
It might be better to render the note on mouse over like in the universe editor.

Reply:
Universe editor is for now Terra incognita for me - will look into it asap ... it may also be useful if pressed `shift` key temporarely reveal all notes ?

Reply:
![screenshot028](https://user-images.githubusercontent.com/54677/151659139-ad0f1ad5-eff6-46be-b01d-444dd6939410.png)

You need to set `devmode = true` in the conf for the editor, but I've attached a screenshot.

Reply:
Mouse over is WIP as of now - I have trouble sorting coordinates calculation.

Reply:
Mouse-over is done; I've added toggling with 'overlay' (tab) key for convenience.